entries/i/interia.pl.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Interia": {
+    "domain": "interia.pl",
+    "url": "https://www.interia.pl/",
```

Comment:
```suggestion
```
The `url` field is not necessary here, as the site is accessible from https://`domain`